build_tools/mako/benchmark_modules_on_android.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Benchmarks modules generated by compile_android_modules.py and writes
+# performance data to a text proto file, e.g., mako-phone-mako_tag-git_hash.
+#
+# The script benchmarks modules on 7-th big core, ie, run with `taskset 80`.
+
+import argparse
+import datetime
+import subprocess
+import os
+import re
+
+import configuration
+
+PATTERN = re.compile(r"BM_(\w+)(.+)/real_time(\s+) (?P<ms>[.0-9]+) ms")
```

Comment:
I still think we should be using the more structured output instead of parsing the human-readable output... but this is ok for now

Reply:
Yes, I think we can let it report a csv or something. I keep this because this is the same behavior as before. I will consider this.